src/components/slideshow/slideshow.js

```diff
@@ -611,6 +614,9 @@ define(['dialogHelper', 'inputManager', 'connectionManager', 'layoutManager', 'f
         self.hide = function () {
             if (dialog) {
                 dialogHelper.close(dialog);
+                // Shows page scrollbar
+                document.body.classList.remove('hide-scroll');
+                document.body.classList.add('force-scroll');
```

Comment:
This should go to `onDialogClosed` because `hide` is for external closing.

Reply:
@dmitrylyzo Done in my last commit. When opening, my approach was correct, right?

Reply:
> When opening, my approach was correct, right?

It depends on when you want to hide scrollbar. As alternative, you could place `classList.add` in promise callback here https://github.com/jellyfin/jellyfin-web/blob/a5b8444e7e982183f94dac274eb5f7fe967cb2fc/src/components/slideshow/slideshow.js#L211-L213
But I didn't check what differs, and it is probably unnoticeable.

Reply:
@dmitrylyzo I just created a photos library and I like how it hides/shows, it's in the right time imho. I think I'll leave it as is.

I tried moving the add/remove to the promise, but it didn't do anything (didn't investigate why tho).

I'll re-request a review now that this is solved and I think nothing else is affected with the new behaviour.